src/views/leaderboard/conponents/TopTrader.tsx

```diff
@@ -46,7 +46,7 @@ export const TopTrader: React.FC<TopTraderProps> = ({ items, loading }) => {
             </p>
           </>
         )}
-        <img src={img2st} className="w-145px xl:w-210px mr--30px xl:mr--50px" />
+        <img src={img2st} className="w-145px xl:w-210px h-66px xl:h-96px mr--30px xl:mr--50px" />
```

Comment:
image size should be set as attribute for browser to render before css loaded

Reply:
right, @ukin-level plz update

Reply:
Thank's @20xlevel. I have updated my pull request.

Reply:
Thank's @20xlevel, I have updated my pull request.

Reply:
it's good to go